reana_job_controller/slurmcern_job_manager.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2019 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CERN Slurm Job Manager."""
+
+import base64
+import os
+from stat import S_ISDIR
+
+from flask import current_app
+
+from reana_job_controller.job_manager import JobManager
+from reana_job_controller.utils import SSHClient
+
+
+class SlurmJobManagerCERN(JobManager):
+    """Slurm job management."""
+
+    SLURM_WORKSAPCE_PATH = ''
+    """Absolute path inside slurm head node used for submission."""
+    REANA_WORKSPACE_PATH = ''
+    """Absolute REANA workspace path."""
+
+    def __init__(self, docker_img=None, cmd=None, env_vars=None, job_id=None,
+                 workflow_uuid=None, workflow_workspace=None,
+                 cvmfs_mounts='false', shared_file_system=False,
+                 job_name=None, kerberos=False):
+        """Instanciate Slurm job manager.
+
+        :param docker_img: Docker image.
+        :type docker_img: str
+        :param cmd: Command to execute.
+        :type cmd: list
+        :param env_vars: Environment variables.
+        :type env_vars: dict
+        :param job_id: Unique job id.
+        :type job_id: str
+        :param workflow_id: Unique workflow id.
+        :type workflow_id: str
+        :param workflow_workspace: Workflow workspace path.
+        :type workflow_workspace: str
+        :param cvmfs_mounts: list of CVMFS mounts as a string.
+        :type cvmfs_mounts: str
+        :param shared_file_system: if shared file system is available.
+        :type shared_file_system: bool
+        :param job_name: Name of the job
+        :type job_name: str
+        """
+        super(SlurmJobManagerCERN, self).__init__(
+            docker_img=docker_img, cmd=cmd,
+            env_vars=env_vars, job_id=job_id,
+            workflow_uuid=workflow_uuid,
+            job_name=job_name)
+        self.compute_backend = "Slurm"
+        self.workflow_workspace = workflow_workspace
+        self.cvmfs_mounts = cvmfs_mounts
+        self.shared_file_system = shared_file_system
+
+    def _transfer_inputs(self):
+        """Transfer inputs to SLURM submit node."""
+        stdin, stdout, stderr = \
+            self.slurm_connection.ssh_client.exec_command('cd /data && pwd')
+        self.slurm_home_path = \
+            stdout.read().decode("utf-8").rstrip()
```

Comment:
Unless I am missing something, isn't `slurm_home_path` always going to be `/data`? If that's the case, wouldn't be this more clear?
```python
self.slurm_home_path = '/data'
stdin, stdout, stderr = \
            self.slurm_connection.ssh_client.exec_command(f'cd {self.slurm_home_path}')
```